packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -354,40 +354,55 @@ module.exports = async ({ models, target }, ctx, { selfFinalize = false } = {})
         }
         case 'belongsToMorph':
         case 'belongsToManyMorph': {
-          const association = definition.associations.find(
-            association => association.alias === name
-          );
+          const association = _.find(definition.associations, { alias: name });
 
-          const morphValues = association.related.map(id => {
-            let models = Object.values(strapi.models).filter(model => model.globalId === id);
+          const morphModelsAndFields = association.related.map(id => {
```

Comment:
here the association.related are models that exists it should be necessary to verify that. consider using getModelByGlobalId instead. Be carefull it must be called of model are mounted (e.g in the relation function)